lib/Common/CommonDefines.h

```diff
@@ -154,8 +154,6 @@
 #ifdef RECYCLER_WRITE_BARRIER
 #if defined(__clang__) 
 #define GLOBAL_FORCE_USE_WRITE_BARRIER 1
-#else
-#define GLOBAL_FORCE_USE_WRITE_BARRIER 1            // force turn on write barrier in windows
```

Comment:
>1  [](start = 39, length = 2)

change to 0 instead of removing it? when we want to test it windows it's easier to turn on